src/components/ebay-tab/test/test.browser.js

```diff
@@ -85,6 +72,52 @@ describe('given tabs with first heading selected', () => {
     }
 });
 
+describe('given tabs with manual activation', () => {
+    const input = mock.Basic_3Headings_3Panels_ActivationManual;
+
+    beforeEach(async() => {
+        component = await render(template, input);
+    });
+
+    describe('when the first heading is activated via keybaord action button', () => {
+        beforeEach(async() => {
+            await pressKey(component.getAllByRole('tab')[0], {
+                key: '(Space character)',
+                keyCode: 32
+            });
+        });
+
+        it('then it does not emit the select event', () => {
+            expect(component.emitted('tab-select')).has.length(0);
+        });
+    });
+
+    describe('when the second tab is activated via keyboard action button', () => {
+        beforeEach(async() => {
+            await pressKey(component.getAllByRole('tab')[1], {
+                key: '(Space character)',
+                keyCode: 32
+            });
+        });
+
+        thenItHasMovedToTab(1);
+    });
+
+    function thenItHasMovedToTab(index) {
```

Comment:
There are 3 `thenItHasMovedToTab` functions, 1 in each major describe block. If they do the same thing, might be good to create a single function that can be used throughout the tests.

Reply:
Sure let me work on that. I didn’t do an optimize sweep, but that’s a fair point.

Reply:
@dkallus Updated.